packages/flutter_tools/lib/src/fuchsia/application_package.dart

```diff
@@ -44,9 +45,10 @@ abstract class FuchsiaApp extends ApplicationPackage {
   File farArchive(BuildMode buildMode);
 }
 
-class PrebuiltFuchsiaApp extends FuchsiaApp {
+class PrebuiltFuchsiaApp extends FuchsiaApp implements PrebuiltApplicationPackage {
   PrebuiltFuchsiaApp({
     required String farArchive,
+    required this.applicationPackage,
   }) : _farArchive = farArchive,
        // TODO(zanderso): Extract the archive and extract the id from meta/package.
        super(projectBundleId: farArchive);
```

Comment:
PrebuiltFuchsiaApp({
    required this.applicationPackage,
  }) : // TODO(zanderso): Extract the archive and extract the id from meta/package.
       super(projectBundleId: applicationPackage.path);